server/config/config_test.go

```diff
@@ -0,0 +1,152 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsValid(t *testing.T) {
+	for _, testCase := range []struct {
+		description string
+		config      *Configuration
+		errMsg      string
+	}{
+		{
+			description: "valid configuration",
+			config: &Configuration{
+				AzureDevopsAPIBaseURL:        "mockAzureDevopsAPIBaseURL",
+				AzureDevopsOAuthAppID:        "mockAzureDevopsOAuthAppID",
+				AzureDevopsOAuthClientSecret: "mockAzureDevopsOAuthClientSecret",
+				EncryptionSecret:             "mockEncryptionSecret",
+			},
+			errMsg: "",
+		},
+		{
+			description: "invalid configuration with AzureDevopsAPIBaseURL empty",
+			config: &Configuration{
+				AzureDevopsAPIBaseURL:        "",
+				AzureDevopsOAuthAppID:        "mockAzureDevopsOAuthAppID",
+				AzureDevopsOAuthClientSecret: "mockAzureDevopsOAuthClientSecret",
+				EncryptionSecret:             "mockEncryptionSecret",
+			},
+			errMsg: "azure devops API base URL should not be empty",
```

Comment:
create constant for this in message.go and use both here as well as in config file